components/event-publisher-proxy/pkg/sender/nats.go

```diff
@@ -7,7 +7,7 @@ import (
 	cenats "github.com/cloudevents/sdk-go/protocol/nats/v2"
 	cev2 "github.com/cloudevents/sdk-go/v2"
 	cev2event "github.com/cloudevents/sdk-go/v2/event"
-	"github.com/nats-io/nats.go"
+	pkgnats "github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/nats"
```

Comment:
Possibly a different name than `pkgnats` as they are all somehow `pkg`. Maybe `epnats` like `cenats` above.

Reply:
"pkgnats" was defined and used in the past, see:
https://github.com/kyma-project/kyma/blob/main/components/event-publisher-proxy/cmd/event-publisher-proxy/nats/nats.go#L13
https://github.com/kyma-project/kyma/blob/main/components/event-publisher-proxy/pkg/handler/nats/handler_test.go#L30
https://github.com/kyma-project/kyma/blob/main/components/event-publisher-proxy/pkg/sender/nats_test.go#L15

Reply:
Okay, so out of scope. 👌🏻